### DISCUSSION TITLE
Weekly report 26.10. - 02.11.

### DISCUSSION THREAD

**Discussion by TadaXD20** [SE-TINF22B6/Plapy#67] [Weekly reports]
# Summary
This week we focused on the database guidelines and on the integration of autogenerated playlists. 
 Before we began two additional steps were needed: 

1. Checking the Software Requirements Specificatons (SRS) whether it is still up to date and if the new features are compatible to them
2. Deepen understanding of the functions by representing them as UML diagrams

## Software Requirements Specification
The latest version can be found here: https://github.com/SE-TINF22B6/Plapy/blob/master/srs.md

## UML diagrams
Our sequence diagram to show the lifecycle of the music bot:
![Sequenzdiagramm_Bot-login drawio](https://github.com/SE-TINF22B6/Plapy/assets/123726628/1ba93f8e-1469-4b4a-b5c4-e673f294a700)

## Other finished tasks
- the bot is able to write to the database and read out information from it
- use of Soundcloud links is now possible

**Comment by Challenger3010** [comment]
Nice and good description of your progress, your UML diagram is understandable and well described so that one can understand everything as an outsider

**Reply by fehlend0** [reply]
Dear Plapy Team,

Great progress for this week!

Your focus on the database guidelines and the integration of autogenerated playlists is a good direction in the development of your project. Moreover, the ability of your bot to read and write to the database is a fundamental functionality, great that you have implemented it. We also find that enabling the use of Soundcloud links is a great extension.

We look forward to hearing more about your project next week!

Your Statify Team
